packages/babel-plugin-transform-react-jsx/src/transform-classic.js

```diff
@@ -19,6 +19,27 @@ export default declare((api, options) => {
   const JSX_ANNOTATION_REGEX = /\*?\s*@jsx\s+([^\s]+)/;
   const JSX_FRAG_ANNOTATION_REGEX = /\*?\s*@jsxFrag\s+([^\s]+)/;
 
+  if ("useSpread" in options) {
+    throw new Error(
+      'transform-react-jsx: Since "useSpread" is always enabled in Babel 8, you must remove it from your config.',
```

Comment:
Should we make these messages a bit nicer for future? Maybe something like:

```suggestion
      'transform-react-jsx: Since Babel 8, "useSpread" is enabled by default. Please remove it from your config.',
```

Reply:
See https://github.com/babel/babel/pull/11141#discussion_r443252782 for wording issues about "enabled by default" and "must" 😄

Reply:
Okay, howabout:

```suggestion
      'transform-react-jsx: Since Babel 8, an inline object with spread elements is always used, and the "useSpread" option is no longer available. Please remove it from your config.',
```